Source/JavaScriptCore/assembler/MacroAssemblerRISCV64.h

```diff
@@ -1752,6 +1752,55 @@ class MacroAssemblerRISCV64 : public AbstractMacroAssembler<Assembler> {
         m_assembler.fmvInsn<RISCV64Assembler::FMVType::D, RISCV64Assembler::FMVType::X>(dest, src);
     }
 
+    /*
+     * These vector instructions are implemented as breaks because the RISC-V V
+     * extension, which will define the instructions they need, is not yet
+     * standardized. They are defined to ensure compilation is possible.
+     */
+    void moveVector(FPRegisterID src, FPRegisterID dest)
+    {
+        UNUSED_PARAM(src);
+        UNUSED_PARAM(dest);
+        m_assembler.ebreakInsn();
+    }
```

Comment:
There's the `MACRO_ASSEMBLER_RISCV64_TEMPLATED_NOOP_METHOD` macro that's usable for this purpose. It doesn't produce a breaking instruction, but it could. But it's useful to quickly cover different overloads for a given function that's not yet supported.